.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,29 @@
+name: Pull Request
+
+on:
+  push:
+    branches: master
+  pull_request:
+    types: [opened, reopened, ready_for_review, synchronize]
+    branches: master
```

Comment:
Could be removed. Otherwise only PRs with `master` as base branch trigger this job.